app/v2/models/des/ReliefClaimed.scala

```diff
@@ -35,7 +35,7 @@ object ReliefClaimed {
   }
 
   case object `CFCSGI` extends ReliefClaimed {
-    override def toTypeOfClaim: TypeOfClaim = TypeOfClaim.`carry-forward-to-carry-sideways`
+    override def toTypeOfClaim: TypeOfClaim = TypeOfClaim.`carry-forward-to-carry-sideways-general-income`
```

Comment:
Why has this changed? Can you check this with Francis please as it is a change from v1 of the api?